pkg/sql/opt/optbuilder/testdata/create_view

```diff
@@ -352,3 +360,21 @@ create-view t.public.v24
  ├── columns: setval:1
  └── dependencies
       └── s
+
+build
+CREATE VIEW v25 AS VALUES ('foo'::foobar)
+----
+create-view t.public.v25
+ ├── VALUES ('foo'::foobar)
+ ├── columns: column1:1
+ └── dependencies
+      └── foobar
+
+build
+CREATE VIEW v26 AS SELECT fb FROM foobars
+----
+create-view t.public.v26
+ ├── SELECT fb FROM t.public.foobars
+ ├── columns: fb:1
+ └── dependencies
+      └── foobars [columns: fb]
```

Comment:
Is it expected that we do not track the dependence on the `foobar` type here? Is this safe because we can determine the transitive dependency of `v26 -> foobars -> foobar`?

Reply:
yeah, i think it's safe since it's not directly referenced by the function. I think the bottom line is that everything should still work if `workday` is dropped/renamed.
That being said, if the UDF references the table `workdays` as an implicit type, then `workday` should be a dependency as well. Could you add a test for that as well? Thanks for fixing the formatting!